Add files via upload

ITSec365 opened the pull request:
New Workbook-AzureThreatResearchMatrixWorkbook

   Required items, please complete
   
   Change(s):
   - New Workbook for Azure Threat Research Matrix 

     Version Updated:
  1.0.0

   Testing Completed:
   - Yes, Tested in Sentinel Environment without custom parsers, functions or tables
  

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes, Tested in Sentinel Environment without custom parsers, functions or tables

v-prasadboke commented:
Hello @ITSec365, Please add workbook images with white and black background. 
Also add workbook metadata to the workbookmetadata file located in workbooks folder

ITSec365 commented:
Hi [v-prasadboke](https://github.com/v-prasadboke), thanks for the response.  Added Images to Images/ Preview Folder and also added my workbook details to workbooksmetadata file.
